### PR TITLE
Fix demos link

### PR DESCRIPTION
**Pull request by matthew-andrews**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Where `opts` is an optional object with properties
     $ make install
     $ make demo
 
-And then visit [http://localhost:8080/demos/local]()
+And then visit [http://localhost:8080/demos/local](http://localhost:8080/demos/local)
```
Comment (ironsidevsquincy) [inline README.md]: no needed, uses the link name as the href

Comment (matthew-andrews) [inline README.md]: but it's broken at the moment

Comment (matthew-andrews) [inline README.md]: https://github.com/Financial-Times/n-video

Comment (ironsidevsquincy) [inline README.md]: quite right, :+1:

**Pull request merged**